test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -500,6 +500,10 @@ class PbsResvAlterError(PtlException):
     pass
 
 
+class PbsSystemError(PtlException):
```

Comment:
I suggest renaming this to `PbsHardwareError` or `PbsRequirementsError` - generally in python SystemError is used when some internal error occurs but here the case is different